sql/hive/src/test/resources/regression-test-SPARK-8489/Main.scala

```diff
@@ -33,8 +33,14 @@ object Main {
   def main(args: Array[String]) {
     // scalastyle:off println
     println("Running regression test for SPARK-8489.")
-    val sc = new SparkContext("local", "testing")
-    val sparkSession = SparkSession.withHiveSupport(sc)
+
+    val conf = new SparkConf()
+      .setMaster("local")
+      .setAppName("testing")
+
+    val sparkSession = SparkSession.builder.enableHiveSupport().getOrCreate()
```

Comment:
this should use the builder's conf method. By the way for this file I would leave it unchanged and fix it in #12924 instead, since @dilipbiswal is actually rebuilding the jar there.